File: abc/abc321/a/main.ts
const main = (input: string) => {
    const n = input.split('').map(s => parseInt(s))
    for (let i = 0; i < n.length - 1; i++) {
      if (n[i] <= n[i + 1]) {
        console.log('No')
        return
      }
    }
    console.log('Yes')
  };
  main(require("fs").readFileSync("/dev/stdin", "utf8").trim());
  

File: abc/abc321/a/main.dart
import 'dart:io';

void main() {
  String N = stdin.readLineSync()!;
  bool is321Like = true;
  
  for (int i = 0; i < N.length - 1; i++) {
    if (N[i] <= N[i + 1]) {
      is321Like = false;
      break;
    }
  }
  
  if (is321Like) {
    print("Yes");
  } else {
    print("No");
  }
}
